framework/decode/dx12_resource_value_mapper.cpp

```diff
@@ -141,6 +148,30 @@ void CopyMappedResourceValuesFromSrcToDst(std::map<uint64_t, T>&
     }
 }
 
+std::string DemangleDxilExportName(const std::string& mangled_name)
+{
+    size_t demangled_name_start = mangled_name.find_first_of("?");
+    size_t demangled_name_end   = mangled_name.find_first_of("@");
+
+    std::string demangled_name;
```

Comment:
Since we check for empty string at the bottom, would it be safer here to explicitly init `demangled_name` to empty?

Reply:
Done